### DISCUSSION TITLE
news/20241106/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#411] [Website Discussions]
# news/20241106/

Introducing Jumper by Witchcraft Software in Sweden. 🇸🇪

https://fcp.cafe/news/20241106/

**Comment by hexeric** [comment]
i think it is purely amazing. and nice to see it's optionally a subscription. note that the AI model needs 2GB RAM, meaning, once you install it and have FCP running, you have 2GB RAM less. the real reason why apple introduced 16GB to be the new minimum. but this is a huge helper especially for solo editors!

**Reply by latenitefilms** [reply]
It shouldn't use much RAM at all after the Analysis is completed.

**Reply by hexeric** [reply]
it does. the helper is always there and uses 2GB (if you load FCP and start the extension or open the jumper app itself). no analysis going on

**Reply by latenitefilms** [reply]
Apologies - you're absolutely correct - I'm seeing the same thing here. I've never really noticed because I'm running 64GB RAM here on my MacBook Pro.

**Reply by maxlund** [reply]
Hi @hexeric, I'm the lead developer of Jumper. You're right that the "JumperHelper" background process takes about 2GB of RAM to run. If we don't keep a lot of things in memory, but instead load them "on-demand", you will have lag  and a slower more unresponsive behaviour when performing searches. All things considered I am quite suprised it only has to take up 2GB of RAM to be honest, given how much VRAM it takes to do things like run local LLMs or local generative image models like StableDiffusion etc.

**Comment by skymakai** [comment]
Is the Non-Subscription option only a 30 search trial?  Is there no other non-sub pricing?

**Reply by latenitefilms** [reply]
You can buy a perpetual/lifetime license for USD$249 - it comes with one year of updates. If you need to update later, you can buy another year of updates for USD$50.

**Comment by jbk111010** [comment]
Just bought it, and playing with it. Wow... Well done, guys

**Comment by jbk111010** [comment]
I wanted to try it out first for the year. Ive been burned on lifetime licenses for software that changes versions, or goes away. 

theworldwidemediaconspiracy.com <http://theworldwidemediaconspiracy.com/>

U.S /France: 646-830-9680 / +33 6 47 69 03 36‬

**Reply by latenitefilms** [reply]
There are options for monthly and yearly subscription, or you can just buy a lifetime/perpetual license - which includes a year of updates. Lots of options!

**Comment by florian-duffe** [comment]
Chris, what a journey. 😅🤗😎
Best of luck to the team! I'm looking forward using it on my next feature. Unfortunatelly next year, though, since I'm stuck in Avid-land.

**Reply by latenitefilms** [reply]
Hopefully Avid support isn't too far away!

**Comment by BacKinnn** [comment]
Does this work well with non real life videos like gaming or animated footage?

**Reply by maxlund** [reply]
Probably, yes. Just try yourself using the free trial here: [getjumper.io](https://getjumper.io)
